GetJobProfiles/ContentRecipes/SharedContent.recipe.json

```diff
@@ -30,7 +30,7 @@
                         }
                     },
                     "GraphSyncPart": {
-                        "Text": "http://nationalcareers.service.gov.uk/sharedcontent/2def38e0-83ae-4bc1-9350-3fb4b00c7857"
+                        "Text": "http://nationalcareers.service.gov.uk/sharedcontent/c0117ac7-115a-4bc1-9350-3fb4b00c7857"
```

Comment:
I think this will need the Content API Prefix replacement code?

Reply:
So it will 😃